### PR TITLE
Typo in DE translation of the NSFW addon

### PR DESCRIPTION
**Pull request by tobiasd**
While "su" is a nice command, it is also a typo at this place.

### CODE REVIEW

File: nsfw/lang/de/messages.po [hunk 1]
```diff
@@ -26,13 +27,13 @@ msgstr "Not Safe for Work (Filter für ungewollte Inhalte) Einstellungen:"
 
 #: nsfw.php:80
```
Comment (MrPetovan) [inline nsfw/lang/de/messages.po]: This isn't right, there is no translated string at this line anymore.

**Comment by annando** [comment]
Oops, I thought it was the opposite ...

**Comment by tobiasd** [comment]
The word plugin will be-occur in many translations again until Transifex got the new sources and they got translated.

File: nsfw/lang/de/messages.po (hunk 1 above)
Comment (tobiasd) [reply nsfw/lang/de/messages.po]: The version of the PHP file from which transifex has the messages.po file had the string in that line. I hope that transifex will soon pick up the new source files, just remembered that you had changed the path from master to develop and transifex was not updated to that information.

**Comment by tobiasd** [comment]
Ok, Transifex has in the mean time picked up the texts with Addon instead of Plugin. I've just translated the DE strings to use the new word. So your automated updates that did not find their way to Transifex should be safe now ;-) I'll update the NSFW addon here in a minute or so.

**Pull request merged**